src/main/resources/net/snowflake/ingest/ingest_error_messages.properties

```diff
@@ -15,7 +15,7 @@
 0010=Missing {0} in config file.
 0011=Failed to upload blob.
 0012=Failed to cleanup resources during {0}.
-0013=Channel {0} is invalid and might contain uncommitted rows, please consider reopening the channel to restart.
+0013=Channel {0} is invalid and might contain uncommitted rows, please consider reopening the channel to restart. Channel invalidated cause: "{1}".
```

Comment:
Should we say `Channel invalidation cause: {1}` ?

Reply:
Make sense. Fixed.